### DISCUSSION TITLE
What versions of postgresql would be expected to work with prefect-server?

### DISCUSSION THREAD

**Discussion by Adam-D-Lewis** [PrefectHQ/prefect#4273] [Q&A]
When running prefect-server locally with an external postgresql database, what versions of postgresql would you expect to function correctly?  I looked in the relevant portion of the docs, but couldn't find this answer.  
https://docs.prefect.io/orchestration/faq/config.html#connecting-to-a-different-api-endpoint

Thanks!

**Comment by cicdw** [comment]
Hi @Adam-D-Lewis - great question; while I haven't personally tested all the various versions, I expect Server to work with postgres 11 and up.  Is there a specific version you are looking into?

**Reply by Adam-D-Lewis** [reply]
Thanks for the response!  I tried out postgres 9.6.2 today, and got a postgres syntax error during the db-upgrade step on the prefect-server-graphql server.  I also tried out postgres 10.16, and it appeared to work fine in the admittedly limited test I ran.

**Reply by cicdw** [reply]
Gotcha, yea that makes sense -- if the migrations run and you can successfully make some queries through the UI then everything should work from there, so it sounds like 10.16 should be fine to use.  We develop on Postgresql 11 so that's a guarantee.
